tvb/tests/library/simulator/integrators_test.py

```diff
@@ -132,6 +132,30 @@ def test_scipy_dop853(self):
     def test_scipy_dopri5(self):
         self._scipy_scheme_tester('Dopri5')
 
+    def test_constrain(self):
+        vode = integrators.VODE(
```

Comment:
why test with VODE?  what do you use in your multiscale stuff?

Reply:
Simpler... Clearer...?

Reply:
Just simpler I guess.

Reply:
I meant why VODE instead of say Euler. But it's OK this doesn't need to change.